### ISSUE TITLE
rss and scatterer not working

### ISSUE THREAD

**Issue by ARSHIAEBRAHIMI** [drobie22/RSS-Reborn-Installer#4]
I installed it using the installer . I had installed already installed the rp1 via ckan . after launching the game , it seems like real solar system and scatterer don't work

**Comment by drobie22** [comment]
Did you launch RP-1 once before running the Reborn installer?

Can you provide your log, see homepage for instructions.

**Comment by ARSHIAEBRAHIMI** [comment]
sure 
[Setup Log 2024-07-20 #006.txt](https://github.com/user-attachments/files/16321735/Setup.Log.2024-07-20.006.txt)

**Comment by drobie22** [comment]
Log seems incomplete. Did you see the finish screen?

Also did you run the game with RP-1 before using the installer

**Comment by ARSHIAEBRAHIMI** [comment]
yes i did finish the screen . i went in game and kerbin was there instead of earth without scatterer 
and also yes i had launched a couple of rockets before using the installer in rp1

**Comment by ARSHIAEBRAHIMI** [comment]
I will run the game again anyway and send the new list as well

**Comment by drobie22** [comment]
> and also yes i had launched a couple of rockets before using the installer in rp1

No like you ran CKAN, ran KSP, saw RP-1 worked, then used RSS-Reborn installer, then everything went back to kerbin? 

This reads like you ran CKAN, then ran the RSS-Reborn installer, then launched the game

**Comment by ARSHIAEBRAHIMI** [comment]
no no . i used ckan to install rp1 yesterday . i even launched a rocket in orbit . today i decided to install rssreborn using the installer. two seperate days , and i had launched the game before using rssreborn installer . It worked properly , I had earth and moon . But now i have kerbin instead

**Comment by drobie22** [comment]
K cool. Just checking. 

Log?

**Comment by ARSHIAEBRAHIMI** [comment]
The log hasn't shown up yet

**Comment by ARSHIAEBRAHIMI** [comment]
[Setup Log 2024-07-20 #006.txt](https://github.com/user-attachments/files/16321761/Setup.Log.2024-07-20.006.txt)
OK here

**Comment by ARSHIAEBRAHIMI** [comment]
Was i supposed to remove the folders mentioned in the manual installation ? or is it done automatically as well?

**Comment by drobie22** [comment]
That's done automatically. First guess is that it deleted what it needed to, then had an error installing the new files. Let me look at the log

**Comment by drobie22** [comment]
> 2024-07-20 20:06:13.413   ComboBoxChange called for ComboBox with Tag: 4
>2024-07-20 20:06:13.413   Length of SizeLabelList: 13
>2024-07-20 20:06:13.413   Length of SizesList: 13
>2024-07-20 20:06:13.413   SizeLabel updated to: Total Size: 350 MB
>2024-07-20 20:06:14.609   Next button clicked, CurPageID: 101
>2024-07-20 20:27:59.368   Message box (Yes/No):
                          Setup is not complete. >If you exit now, the program will not be installed.
                          
>                          You may run Setup again at another time to complete the installation.
                          
                          Exit Setup?
>2024-07-20 20:28:00.881   User chose Yes.

Looks like the installer was manually quit halfway through

**Comment by ARSHIAEBRAHIMI** [comment]
That wasn,t done by me . i left my computer and just came back . should rerun the installer ? 
Does it have to download the files again or it has them cached?

**Comment by ARSHIAEBRAHIMI** [comment]
I even recieved the installation complete message if I recall correctly

**Comment by drobie22** [comment]
Caching is not a feature (yet). Definitely re run the latest installer, and make sure you watch the screen change from the download page to the extraction page. That'll confirm a few things

**Comment by ARSHIAEBRAHIMI** [comment]
sure thanks

**Comment by ARSHIAEBRAHIMI** [comment]
also , what does the include sc mean ?

**Comment by drobie22** [comment]
Scaled? It's a different way of rendering the selected body but it impacts performance. I would hold off for now

**Comment by ARSHIAEBRAHIMI** [comment]
I tried again . same problem . 
![Screenshot 2024-07-20 221655](https://github.com/user-attachments/assets/1aa64e0a-e66a-45c2-bb18-07ef0e6bc39b)
![Screenshot (30)](https://github.com/user-attachments/assets/80762710-d84e-42cd-9125-df411940c4ae)

**Comment by ARSHIAEBRAHIMI** [comment]
[Setup Log 2024-07-20 #007.txt](https://github.com/user-attachments/files/16322074/Setup.Log.2024-07-20.007.txt)
here's the setup log

**Comment by drobie22** [comment]
Log looks great. Do you have multiple directories or installs?

**Comment by ARSHIAEBRAHIMI** [comment]
No just this one instance

**Comment by ARSHIAEBRAHIMI** [comment]
Should the directory be a in a steam folder ?

**Comment by drobie22** [comment]
As long as your KSP exe is in C:\Users\Arshia Ebrahimi\Desktop\Games\Modding\Kerbal Space Program it should work. Try launching the KSP exe directly from that folder

**Comment by ARSHIAEBRAHIMI** [comment]
Didn't work again

**Comment by KerballOne** [comment]
I too am experiencing the same symptoms. 
I have been playing this RP-1 instance for months, so it definitely satisfies (ran RSS prior to running the Reborn installer).

After running the installer, no errors, says finished.
When game loaded, RSS is not there, planet is Kerbin, and all my vessels around and on the Moon are now on Laythe.
Also, got a bunch of Contract Configurator error popups about contracts failed (my guess is they were dependent on RSS bodies).

**Comment by drobie22** [comment]
At this point try doing a manual install following fox's instructions and see if that works

**Comment by drobie22** [comment]
> I too am experiencing the same symptoms. 
> 

If you both don't mind, can I get your latest installer log as well as the latest KSP logs? 

I'll try and dig into this tonight. Very interesting.

**Comment by KerballOne** [comment]
First step in Fox's instructions, is to remove some folders from GameData.  Noticed RealSolarSystem isn't there.
I'll have to upload the big KSP.log to dropbox for you.  Where is the installer log?

**Comment by drobie22** [comment]
Instructions for logs are in the read me. Seems like the installer is deleting the files fine so part of it works lol. In the next patch I might have the log list gamedata folders as verification

**Comment by KerballOne** [comment]
https://www.dropbox.com/scl/fo/kojhvxw3652ade6e8a8ny/AN6_-OyYgg_86230lKQ_Ys8?rlkey=f6qpiwdwv3ugoftmg1dybq828&st=y44ijbag&dl=0

Thanks.  I can see the "deleted RealSolarSystem" log entry, but not one to create.

**Comment by ARSHIAEBRAHIMI** [comment]
I manually replaced ballistic fox's kopernicus , rssve and rss config files in the game data and it finally worked 
seems like the issue is in one of these 3 mods

**Comment by KerballOne** [comment]
I'm a bit confused.  there is no RealSolarSystem folder or RealSolarSystem.dll anymore.  Fox's instruction say to delete it, but not replace it with anything.  The configs are there, but what actually provides the mod?

**Comment by drobie22** [comment]
RSS Reborn itself is a fork of Real Solar System. It's split between RSS Configs, terrain, textures, etc.

**Comment by KerballOne** [comment]
![image](https://github.com/user-attachments/assets/28993bf8-4780-462d-ac70-69a2cb21921b)
Still Kerbin, and now in a crater

**Comment by KerballOne** [comment]
Thanks.  I looked for the dll and didn't find it.  Ran the installer again, it did not create the RSS-Configs folder at all.  Following Fox's manual install step 5 for RSS-Configs, so the RealSolarSystem.dll is in there under plugins,... and trying to start the game.  

It was a bit confusing with the change in file structure, usually a "configs" folder doesn't have the plugin dlls, so that was a change I didn't consider.
